br-webui/style.css

```diff
@@ -36,3 +36,71 @@ body {
 .slider {
   width: 100% !important;
 }
+
+.switch {
+  position: relative;
+  top: 7px;
+  display: inline-block;
+  width: 30px;
+  height: 15px;
+}
+
+.heading {
+  margin-top: 30px;
+}
+
+/* Hide default HTML checkbox */
+.switch input {display:none;}
+
+.slidernew {
+  position: absolute;
+  cursor: pointer;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-color: #ccc;
+  -webkit-transition: .4s;
+  transition: .4s;
+}
+
+.slidernew:before {
+  position: absolute;
+  content: "";
+  height: 10px;
+  width: 10px;
+  left: 3px;
+  bottom: 2.5px;
+  top: 2.4px;
+  background-color: white;
+  -webkit-transition: .4s;
+  transition: .4s;
+}
+
+input:checked + .slidernew {
+  background-color: #2196F3;
+}
+
+input:focus + .slidernew {
+  box-shadow: 0 0 1px #2196F3;
+}
+
+input:checked + .slidernew:before {
+  -webkit-transform: translateX(26px);
+  -ms-transform: translateX(26px);
+  transform: translateX(14px);
+}
+
+/* Rounded sliders */
+.slidernew.round {
+  border-radius: 34px;
+}
+
+.slidernew.round:before {
+  border-radius: 50%;
+}
+
+.toggle {
+  margin-left: 904px;
```

Comment:
why is this here? how did you come up with this number?

Reply:
I saw the alignment and put this number according the position i wanted it to be in. It doesnt move even if I zoom the page in and out.